app/src/main/java/mozilla/lockbox/store/AccountStore.kt

```diff
@@ -163,15 +163,15 @@ open class AccountStore(
         val syncCredentialSubject = syncCredentials as Subject
         val fxa = fxa ?: return
         securePreferences.putString(Constant.Key.firefoxAccount, fxa.toJSONString())
-
-        fxa.getProfile()
-            .asSingle(coroutineContext)
+        
+        fxa.getProfileAsync()
+            .asMaybe(coroutineContext)
```

Comment:
<img width="595" alt="image" src="https://user-images.githubusercontent.com/43795363/59286980-99970280-8c25-11e9-8e8a-fc6e531b8378.png">

Is `asMaybe` the correct thing to use here?

Reply:
yes!